planex/Makefile.rules

```diff
@@ -101,7 +101,7 @@ $(TOPDIR)/MANIFESTS/%.json:
 $(TOPDIR)/SOURCES/%:
 	@echo [FETCH] $@
 	$(AT) mkdir -p $(@D)
-	$(AT)$(FETCH) $(FETCH_FLAGS) $< $@
+	$(AT)$(FETCH) $(FETCH_FLAGS) $^ $@
```

Comment:
What does `$^` mean?

Reply:
All dependencies, rather than the leftmost dependency

Reply:
https://www.gnu.org/software/make/manual/make.html#Automatic-Variables

Reply:
Thanks!